.github/workflows/test-publish-to-npm.yml

```diff
@@ -0,0 +1,17 @@
+on: push
```

Comment:
Misschien dat we deze job gewoon weg kunnen laten

Reply:
Verwijderd